bundles/binding/org.openhab.binding.ddwrt/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,33 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: openHAB ddwrt Binding
```

Comment:
The [official name](https://www.dd-wrt.com/site/) seems to be spelled `DD-WRT`.  Would you mind using that in textual areas?